github/github.go

```diff
@@ -676,3 +676,33 @@ func (client *Client) AllRepositories(user *github.User) ([]*github.Repository,
 
 	return ownedRepos, nil
 }
+
+func (client *Client) CreateLabels(repo *github.Repository) error {
+	updateMeLabel := config.UpdateMeLabels[0]
+	doNotMergeLabel := config.DoNotMergeLabels[0]
+	mergeWhenReadyLabel := config.MergeWhenReadyLabels[0]
```

Comment:
Right now, it looks like if there are multiple labels that are accepted for each condition, only the first one is checked/created here. If a repository already has the secondary labels, then this will lead to "duplicate" labels (different labels that have the same behavior).

I think we need to either:

1.  Account for this by checking for all possible labels, and only creating the first one if none exist
2. Change the configuration so that there can only be one label for each condition